File: Blackjack21-JAVA/Blackjack21Library/src/Blackjack21/Game/Exceptions/PlayerHandNotSplitException.java
package Blackjack21.Game.Exceptions;

/**
 * Throws an exception if the player's hand is not split
 */
public class PlayerHandNotSplitException extends Exception
{

    public PlayerHandNotSplitException()
    {
        super("The player's hand is not split");
    }

    public PlayerHandNotSplitException(String message)
    {
        super(message);
    }

}

File: BlackJack-CSHARP/Blackjack21.Game/Game/Exceptions/PlayerHandNotSplitException.cs
using System;

namespace Blackjack21.Game.Exceptions
{
    /// <summary>
    /// Throws an exception if the player's hand is not split
    /// </summary>
    public class PlayerHandNotSplitException : Exception
    {

        public PlayerHandNotSplitException(string message = "The player's hand is not split") : base(message)
        {

        }
    }
}
